src/features/device-state/routes/state-patch-v3.ts

```diff
@@ -319,6 +330,22 @@ export const statePatchV3: RequestHandler = async (req, res) => {
 							deviceBody.is_running__release = release.id;
 						}
 					}
+					const { releases } = apps[userAppUuid];
```

Comment:
Since `userAppUuid = device.belongs_to__application[0].uuid` should we first check whether `apps[userAppUuid]` is not null before destructuring it?
```suggestion
					const releases = apps[userAppUuid]?.releases;
```

Reply:
So interestingly this change you proposed triggers an error
```
src/features/device-state/routes/state-patch-v3.ts:321:11 - error TS7022: 'device' implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer.
```
@thgreasi, I'm not sure why though